eng/pipelines/VS-release.yml

```diff
@@ -1,8 +1,10 @@
 ---
 name: $(Date:yyyMMdd).$(Rev:r)
 jobs:
-- template: ./jobs/VSEng-MicroBuildVS2019.job.yml
+- template: ./jobs/VSEngSS-MicroBuild2019.job.yml
   parameters:
-    JobName: VS_Release
-    BuildTemplate: 'VS-release'
+    DisplayName: 'VSCode_Release'
```

Comment:
>VSCode_Release' [](http://example.com/codeflow?start=18&length=15)

Did you want this change?

Reply:
Whoops, bad copy and paste for the display name.